src/collections/match/operations/match-lock.operation.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  BlapiResponse,
+  BlError,
+  Match,
+  MatchVariant,
+} from "@boklisten/bl-model";
+import { ObjectId } from "mongodb";
+
+import { getAllMatchesForUser } from "./match-operation-utils";
+import { isBoolean } from "../../../helper/typescript-helpers";
+import { Operation } from "../../../operation/operation";
+import { BlApiRequest } from "../../../request/bl-api-request";
+import { BlDocumentStorage } from "../../../storage/blDocumentStorage";
+import { BlCollectionName } from "../../bl-collection";
+import { matchSchema } from "../match.schema";
+
+export interface MatchLockSpec {
+  customer: string;
+  lock: boolean;
+}
+
+export function verifyMatchLockSpec(
+  matchLockSpec: unknown,
+): matchLockSpec is MatchLockSpec {
+  const m = matchLockSpec as Record<string, unknown>;
```

Comment:
```suggestion
  const m = matchLockSpec as Record<string, unknown> | null | undefined;
```
and maybe extract a type for this as we should be using this type more generally